lib/copyable/copyable_extension.rb

```diff
@@ -59,6 +59,13 @@ def copyable(&block)
               # for this brand new model, visit all of the associated models,
               # making new copies according to the instructions in the copyable
               # declaration
+
+              if options[:skip_associations].present?
+                # To skip an association, change its advice to :do_not_copy
+                options[:skip_associations].each do |association|
+                  main.association_list[association] = :do_not_copy if main.association_list.keys.include?(association)
+                end
+              end
```

Comment:
It looks like `main.association_list` is already passed to `Declarations::Associations#execute` (as is the `:skip_validations` option, showing that passing options to it is a thing). Would it make sense for this logic to live there, instead?